maple-sdk/introduction.md

```diff
@@ -0,0 +1,31 @@
+# Introduction
+
+`maple-js` is the JavaScript SDK for Maple Protocol, a comprehensive and user-friendly toolkit designed to facilitate seamless interaction with Maple Protocol's blockchain-based smart contracts. This SDK provides developers with the necessary tools to integrate with the Maple Protocol ecosystem and it simplifies the process of connecting to the blockchain, managing accounts, submitting transactions, and handling events generated by smart contracts.
+
+By utilizing the JavaScript SDK for Maple Protocol, developers can efficiently create robust and secure dApps that leverage the full potential of Maple Protocol's smart contracts, fostering innovation and driving the adoption of decentralized finance solutions.
+
+## Features
+
+1. **Collateral management** through `collateralLockerImports`
```

Comment:
All of these are imported as `xImports` but they are exported just as `x`... so `collateralLocker` rather than `collateralLockerImports`